### PR TITLE
chore: v3 request reply support

### PR DESCRIPTION
**Pull request by GreenRover**
**Description**

Update v3 models:
- fix models from https://github.com/asyncapi/spec/pull/847
- add unit tests

**Related issue(s)**
See also #732

### CODE REVIEW

**Comment by GreenRover** [comment]
Now the pr #708 gots merged. I rebased my pr and only add parts where i not agree with #708 
@magicmatatjahu Please review

**Comment by GreenRover** [comment]
Ping @magicmatatjahu

File: test/models/v3/operation.spec.ts [hunk 1]
```diff
@@ -150,6 +150,20 @@ describe('Operation model', function() {
     });
   });
 
+  describe('.reply()', function() {
```
Comment (magicmatatjahu) [inline test/models/v3/operation.spec.ts:153]: In this file there're already tests for this method. Remove one of them.

File: src/models/v3/operation.ts [hunk 2]
```diff
@@ -89,4 +89,13 @@ export class Operation extends OperationTrait<v3.OperationObject> implements Ope
...
   }
+
+  hasReply(): boolean {
+    return !!this._json.reply;
+  }
+
+  reply(): OperationReplyInterface | undefined {
+    if (!this._json.reply) return undefined;
+    return new OperationReply(this._json.reply);
+  }
```
Comment (magicmatatjahu) [inline src/models/v3/operation.ts:100]: There is already a method like that in this class.

File: src/models/operation.ts [hunk 3]
```diff
@@ -17,4 +18,6 @@ export interface OperationInterface extends BaseModel, OperationTraitInterface {
   messages(): MessagesInterface;
   reply(): OperationReplyInterface | undefined;
   traits(): OperationTraitsInterface;
+  hasReply(): boolean;
+  reply(): OperationReplyInterface | undefined;
```
Comment (magicmatatjahu) [inline src/models/operation.ts:22]: There's already a method like that in this interface.

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]

File: src/models/operation.ts (hunk 3 above)
Comment (GreenRover) [reply src/models/operation.ts:22]: done

File: src/models/v3/operation.ts (hunk 2 above)
Comment (GreenRover) [reply src/models/v3/operation.ts:100]: done

File: test/models/v3/operation.spec.ts (hunk 1 above)
Comment (GreenRover) [reply test/models/v3/operation.spec.ts:153]: done

**Comment by GreenRover** [comment]
please re review

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**